pkg/devices/gen_net.go

```diff
@@ -82,8 +82,13 @@ func NewGenNetDevice(deviceID string, dt types.DeviceType, isRdma bool) (*GenNet
 	}
 
 	linkType := ""
-	if len(ifName) > 0 {
-		la, err := utils.GetNetlinkProvider().GetLinkAttrs(ifName)
+	linkTypeProviderDevice := ifName
```

Comment:
Hey,

can you add a test for this one ?

Reply:
Done